src/main/services/liv/liv.service.ts

```diff
@@ -25,7 +25,8 @@ export class LivService {
             win32: async () => {
                 const regRes = await regedit.promisified.list([this.livRegeditKey]).then(res => res[this.livRegeditKey]);
                 return regRes?.exists;
-            }
+            },
+            linux: async() => false,
```

Comment:
I think it should be better to pass `true` to the `execOnOs` function to not throw error (see function signature)
At least it will handle MacOs as well

indeed I will soon add e2e testing that will run on windows/linux/macos so if e2e tests was already there your PR not pass the checks

Reply:
Passing true makes it return undefined which causes an error when calling `then` on the result.

Reply:
in that case i think you can convert `isLivInstalled` to an async function and it will return a promise in any case

Reply:
Yes, that works. That's an interesting bit of behavior I wasn't aware of, good to know!